modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -79,8 +79,7 @@ a| Use one of the following values to specify if the admission controller must c
 The default value is `DoNotScanInline`.
 
 | `admissionControl.timeoutSeconds`
-| Use this parameter to specify the maximum number of seconds {product-title} must wait for an admission review before marking it as fail open.
-
+| Use this parameter to specify the maximum number of seconds {product-title-short} must wait for an admission review before marking it as fail open. Beginning in release 4.5, Red{nbsp}Hat reduced the default timeout setting for the {product-title-short} admission controller webhooks from 20 seconds to 10 seconds, resulting in an effective timeout of 12 seconds within the `ValidatingWebhookConfiguration`. This change does not negatively affect {ocp} users because {ocp} caps the timeout at 13 seconds.
```

Comment:
Existing phrasing, but does "fail open" make sense to the expected reader (or is it explained earlier)?

Reply:
That's a good question - it is an existing term and I don't see it anywhere else. I'll see if I can find out!

Reply:
Just for the record, the official Kubernetes documentation on this topic can be found here: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy

They use `fail-open` (with a dash) in these docs.

Reply:
OK, based on some slack research and the link above for the K8s docs, I've added a short description that I hope is accurate. I looked and old OpenShift docs use "fail open" (no hyphen) so I guess I will match those.

Reply:
reads well to me - looks like the PR needs a refresh